oak/server/rust/oak_abi/build.rs

```diff
@@ -15,11 +15,17 @@
 //
 
 fn main() {
-    oak_utils::run_protoc_rust(protoc_rust::Args {
-        out_dir: "src/proto",
-        input: &["../../../../oak/proto/oak_api.proto"],
-        includes: &["../../../../oak/proto"],
-        customize: protoc_rust::Customize::default(),
-    })
-    .expect("protoc");
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=../../../../oak/proto/oak_api.proto");
+
+    let mut config = prost_build::Config::new();
+    config.out_dir("src/proto");
+    config
```

Comment:
I think `out_dir` and `compile_protos` can be chained all together (and avoid the `config` mut binding altogether).